planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/src/scene_module/intersection/scene_intersection.cpp

```diff
@@ -297,7 +302,7 @@ double IntersectionModule::calcIntersectionPassingTime(
   double dist_sum = 0.0;
   int assigned_lane_found = false;
 
-  for (int i = closest_idx + 1; i < path.points.size(); ++i) {
+  for (size_t i = closest_idx + 1; i < path.points.size(); ++i) {
```

Comment:
+1 for size_t. I have seen in apex code we mostly use std::size_t but that seems pedantic

Reply:
Yep, good thing we enabled all the warnings and GCC told me about the int to size_t comparisons.